pg-index-health-core/src/test/java/io/github/mfvanek/pg/checks/host/ColumnsWithSerialTypesCheckOnHostTest.java

```diff
@@ -44,9 +46,9 @@ void onDatabaseWithThem(final String schemaName) {
                 .hasSize(2)
                 .containsExactly(
                     ColumnWithSerialType.ofBigSerial(
-                        Column.ofNotNull(ctx.enrichWithSchema("bad_accounts"), "real_account_id"), String.format("%s.bad_accounts_real_account_id_seq", schemaName)),
+                        Column.ofNotNull(ctx.enrichWithSchema("bad_accounts"), "real_account_id"), String.format(Locale.ROOT, "%s.bad_accounts_real_account_id_seq", schemaName)),
```

Comment:
It looks like we can add a new method `enrichSequenceWithSchema` to the `PgContext`.
It can be like `ctx.enrichSequenceWithSchema("bad_accounts_real_client_id_seq")`

Reply:
I've added a new issue for that https://github.com/mfvanek/pg-index-health/issues/422